client/lib/products-values/index.js

```diff
@@ -166,13 +168,27 @@ export function isJetpackMonthlyPlan( product ) {
 	return isMonthly( product ) && isJetpackPlan( product );
 }
 
-export function isMonthly( product ) {
-	product = formatProduct( product );
+export function isMonthly( rawProduct ) {
+	const product = formatProduct( rawProduct );
 	assertValidProduct( product );
 
 	return parseInt( product.bill_period, 10 ) === PLAN_MONTHLY_PERIOD;
 }
 
+export function isYearly( rawProduct ) {
+	const product = formatProduct( rawProduct );
+	assertValidProduct( product );
+
+	return parseInt( product.bill_period, 10 ) === PLAN_ANNUAL_PERIOD;
+}
+
+export function isBiennially( rawProduct ) {
+	const product = formatProduct( rawProduct );
+	assertValidProduct( product );
+
+	return parseInt( product.bill_period, 10 ) === PLAN_BIENNIAL_PERIOD;
+}
+
```

Comment:
just for fun there's a possible iteration on this we could take to reuse some code…

```js
const matchesPeriod = period => rawProduct => {
	const product = formatProduct( rawProduct );
	assertValidProduct( product );

	return period === parseInt( product.bill_period, 10 );
}

export const isMonthly = matchesPeriod( PLAN_MONTHLY_PERIOD );
export const isYearly = matchesPeriod( PLAN_ANNUAL_PERIOD );
export const isBiennially = matchesPeriod( PLAN_BIENNIAL_PERIOD );
```

Reply:
good idea!